app/src/main/resources/application.properties

```diff
@@ -111,6 +111,7 @@ registry.limits.config.cache.check-period=30000
 
 registry.auth.enabled=${AUTH_ENABLED:false}
 registry.auth.owner-only-authorization=${AUTHZ_ENABLED:false}
+registry.auth.roles.enabled=${ROLES_ENABLED:false}
```

Comment:
This is a breaking change with the previous behaviour. I think we want to make this true by default

Reply:
I disagree. It is not really a breaking change since auth is disabled by default so in order to get the previous behaviour you just need to also set this to true. If it's not strictly required, I usually prefer to have things disabled by default.

Reply:
I can see both points of view here, for sure.  It seems like a breaking change because it is a change in behavior.  But it's a more permissive change in behavior, so it's not technically a breaking change.  :)

I tend to agree with @carlesarnal in this case - it should be disabled by default.